tests/Microsoft.Identity.Test.Unit/PublicApiTests/ClientCredentialWithRegionTests.cs

```diff
@@ -329,6 +329,89 @@ public async Task MsalForceRegionIsSet_RegionIsUsed()
             }
         }
 
+        [TestMethod]
+        public async Task MsalDisableRegionEnvVariable_API()
+        {
+            using (new EnvVariableContext())
+            using (var harness = base.CreateTestHarness())
+            {
+                Environment.SetEnvironmentVariable(
+                    ConfidentialClientApplicationBuilder.DisableRegionEnvVariable, "1");
+
+                var httpManager = harness.HttpManager;
+
+                var cca = ConfidentialClientApplicationBuilder
+                                .Create(TestConstants.ClientId)
+                                .WithHttpManager(httpManager)
+                                .WithAzureRegion(TestConstants.Region)
+                                .WithClientSecret(TestConstants.ClientSecret)
+                                .Build();
+
+                await ExpectNoRegionAsync(httpManager, cca).ConfigureAwait(false);
+            }
+        }
+
+        [TestMethod]
+        public async Task MsalDisableRegionEnvVariable_OtherEnv()
+        {
+            using (new EnvVariableContext())
+            using (var harness = base.CreateTestHarness())
+            {
+                Environment.SetEnvironmentVariable(
+                    ConfidentialClientApplicationBuilder.ForceRegionEnvVariable, "someregion");
```

Comment:
Do we have a const defined already for test regions?

Reply:
I find the tests to be more readable without constants. I don't see any value in constants in test code.